src/please.go

```diff
@@ -268,6 +268,11 @@ var opts struct {
 		} `command:"pleasings" description:"Initialises the pleasings repo"`
 		Pleasew struct {
 		} `command:"pleasew" description:"Initialises the pleasew wrapper script"`
+		Plugin struct {
+			Args struct {
+				Plugins []string `positional-arg-name:"plugin" required:"true" description:"Plugins to install"`
+			} `positional-args:"true"`
+		} `command:"plugin" description:"Install a plugin and migrate any language-specific config values"`
```

Comment:
As discussed IRL, it might make sense to hide this until v17. There are also other issues with plugins that need fixing meanwhile. Adding `hidden:"true"` after `command:"plugin"` should do it.